ios/BitcoinJungle.xcodeproj/project.pbxproj

```diff
@@ -577,7 +577,7 @@
 				COPY_PHASE_STRIP = YES;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
this seems to be a necessary change to have ios build on xcode 12.

> The root reason is that Xcode 12 does not allow build arm64 architecture for Simulator. We should set x86_64 for Simulator building.

https://stackoverflow.com/questions/63993084/problems-after-upgrading-to-xcode-12ld-building-for-ios-simulator-but-linking

Reply:
Interesting. I'm running a M1 Mac (ARM) with Xcode 13.1 (macOS Monterey 12.0.1), so I hadn't noticed this problem yet. I'll have to try it out here to see what it does on my machine. Maybe this should just be a developer note if they're using an older version of Xcode?

Reply:
i just removed this, it seems to work fine without it.